README.md

```diff
@@ -120,6 +120,21 @@ In the Run configurations, copy the PHPUnit configuration and use these settings
 2. right-click on `Tests/Acceptance/StarterCest.php`
 3. Run 'Acceptance (Codeception)'
 
+## Developing locally
+
+In order to work on the extension locally, you can use a local environment (local PHP, server) or use
+a widely adopted tool in TYPO3 Community, [ddev](https://github.com/drud/ddev), which we recommend.
+
+## Running Composer commands in the DDEV container
```

Comment:
We also need to document how to build the DDEV setup without starting the webserver. (`ddev start` isn't the right command AFAICT.)

Reply:
And we still need to solve the port problem. If I have a local Apache on port 80, I cannot run any ddev command:

```
klee@gonzales ~/src/typo3/ext/tea (TASK/Add-base-ddev-command)$ ddev composer ci:ts:lint
Building ddev-ssh-agent 
Creating ddev-ssh-agent ... done
 
ssh-agent container is running: If you want to add authentication to the ssh-agent container, run 'ddev auth ssh' to enable your keys. 
Pushed mkcert rootca certs to ddev-global-cache/mkcert 
Running   Command=ip address show dev docker0
Building db 
Building web 
Creating ddev-tea-db ... done
Creating ddev-tea-dba ... done
Creating ddev-tea-web ... done
 
Failed to start tea: Unable to listen on required ports, port 80 is already in use,
Troubleshooting suggestions at https://ddev.readthedocs.io/en/stable/users/troubleshooting/#unable-listen 
```

Reply:
Port solved. 
I don't get what do you mean with `ddev start` command. Running ddev composer ci:ts:lint should be enough to start containers and run lint

Reply:
Ah, it seems there is no need to initialize anything beforehand, and I can use commands like `ddev composer ci:ts:lint` directly. Nice!